### PR TITLE
Disabled option section when collapsed mode not checked per #45993

### PR DESCRIPTION
**Pull request by ismailsunni**
## Description

Fix the first item from #45993:
> we should either grey out or hide the "Options" section if "collapsed" mode is not selected. And remove the "collapse only" mention from the title. I don't think there's any place we do this

Peek 2022-02-02 15-41 - disabled collapsed legend options

Funded by Camptocamp https://www.camptocamp.com

### CODE REVIEW

**Review by alexbruy — APPROVED** [review]

File: src/ui/qgsdatadefinedsizelegendwidget.ui [hunk 1]
```diff
@@ -218,6 +218,7 @@
  </tabstops>
  <resources>
   <include location="../../images/images.qrc"/>
+  <include location="../../images/images.qrc"/>
```
Comment (agiudiceandrea) [inline src/ui/qgsdatadefinedsizelegendwidget.ui:221]: Is this needed or is it a typo?

Comment (ismailsunni) [reply src/ui/qgsdatadefinedsizelegendwidget.ui:221]: Hmm, I edited it using Design Mode in Qt Creator. Perhaps it's a bug from the Qt Creator? I will delete the duplicate manually then.

I checked just now, there is a lot of this kind of issue in the other files (e.g. https://github.com/qgis/QGIS/blob/master/src/ui/qgsstatisticalsummarybase.ui#L174)

Comment (agiudiceandrea) [reply src/ui/qgsdatadefinedsizelegendwidget.ui]: > Hmm, I edited it using Design Mode in Qt Creator. Perhaps it's a bug from the Qt Creator? I will delete the duplicate manually then.
> 
> I checked just now, there is a lot of this kind of issue in the other files (e.g. https://github.com/qgis/QGIS/blob/master/src/ui/qgsstatisticalsummarybase.ui#L174)

You are right. It seems to me there are 17 .ui files with more than one `  <include location="../../images/images.qrc"/>` lines.

Comment (nyalldawson) [reply src/ui/qgsdatadefinedsizelegendwidget.ui]: >Perhaps it's a bug from the Qt Creator?

Yeah it is -- it randomly just duplicates these lines for some reason

Comment (nyalldawson) [reply src/ui/qgsdatadefinedsizelegendwidget.ui]: (they are harmless, but annoying)

Comment (agiudiceandrea) [reply src/ui/qgsdatadefinedsizelegendwidget.ui]: https://github.com/qgis/QGIS/pull/47170

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**